[
 {
  "author": "stefano", 
  "date": "2003-03-25T15:11:32.000000Z", 
  "msg": [
   "refactored flow samples"
  ], 
  "revision_id": "23879"
 }, 
 {
  "author": "stefano", 
  "date": "2003-03-25T14:25:18.000000Z", 
  "msg": [
   "NOTE: back-incompatible change!!! (affect only users currently using the flow)", 
   "Changed the URL called when an invalid continuation is called from \"invalidContinuation.xml\" to a more semantically neutral \"invalidContinuation\".", 
   "Consider this a half-fix since I think the flow engine should raise an exception and let it handle by the normal error-handling subsystem."
  ], 
  "revision_id": "23878"
 }, 
 {
  "author": "stefano", 
  "date": "2003-03-25T13:53:56.000000Z", 
  "msg": [
   "removed samples, improved navigation, fixed DTD reference"
  ], 
  "revision_id": "23877"
 }, 
 {
  "author": "stefano", 
  "date": "2003-03-25T13:48:11.000000Z", 
  "msg": [
   "added by mistake"
  ], 
  "revision_id": "23876"
 }, 
 {
  "author": "stefano", 
  "date": "2003-03-25T13:44:37.000000Z", 
  "msg": [
   "moving flow documents into the xdocs (where they belong)"
  ], 
  "revision_id": "23875"
 }
]